### PR TITLE
Docs/improve documentation

### PR DESCRIPTION
**Pull request by sarah-segel**

### CODE REVIEW

File: docs/guides/first-steps.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # First Steps
+
+Once the requirements for both an optimizer and a benchmark, e.g. `SMAC2.0` and `BBOB`, 
+are installed, you can run one of the following minimal examples to benchmark 
+`SMAC2.0` on `BBOB` directly with Hydra:
```
Comment (benjamc) [inline docs/guides/first-steps.md:5]: add install commands again

File: docs/contributing/contributing-an-optimizer.md [hunk 2]
```diff
@@ -1 +1,63 @@
 # Contributing an Optimizer
+
+To add a new optimizer to CARPS, you need to create a new Python file that defines a new optimizer 
+class. This class should inherit from the `Optimizer` class defined in 
+`carps/optimizers/optimizer.py`. Here's a step-by-step guide:
+
```
Comment (benjamc) [inline docs/contributing/contributing-an-optimizer.md:6]: add reference to template repo https://github.com/automl/CARP-S-template/blob/main/my-optimizer.py

**Review by benjamc — CHANGES_REQUESTED** [review]

File: docs/contributing/contributing-an-optimizer.md (hunk 2 above)
Comment (sarah-segel) [reply docs/contributing/contributing-an-optimizer.md:10]: Added.

File: docs/guides/first-steps.md (hunk 1 above)
Comment (sarah-segel) [reply docs/guides/first-steps.md]: Right, I added the link to the install instructions (to not have the instructions itself in two places).

**Pull request merged**